web/themes/custom/hatter/templates/page.html.twig

```diff
@@ -71,8 +72,14 @@
   </section>
 {% endif %}
 
-{% if page.footer|render|trim is not empty %}
+{% if page.footer_left or page.footer_right %}
   <footer class="site-footer">
-    {{ page.footer }}
+    <div class="container">
+      <div class="l-2up--1">{{ page.footer_left }}</div>
+      <div class="l-2up--2">{{ page.footer_right }}</div>
+      <div class="l-1up">
+        <small class="footer-copyright">Copyright 2017 Midcamp Lorem ipsum dolor sit amet, consectetur adipisicing.</small>
```

Comment:
We've actually never had a copyright on other sites. I'd recommend just leaving this out.